settings.py

```diff
@@ -214,6 +215,8 @@ def __assure_sf2_exists__(self):
             ['/usr/local/share/soundfonts'],
             [xdg.DATA_HOME, 'sounds/sf2'],
             [xdg.DATA_HOME, 'soundfonts'],
+            [self.distdir, 'share/sounds/sf2'],
+            [self.distdir, 'share/soundfonts'],
```

Comment:
When I was writing this code, I forgot about the existence of `XDG_DATA_DIRS` and I intend to replace this directory list at some point; on my system it already contains `~/.local/share/flatpak/exports/share/`, so maybe it would be better place to look for sf2 files? I don't know if fluid or other sound fonts are packaged using Flatpak already…

Reply:
`self.distdir` here points to steam-dos installation, so the path expands to `compatibilitytools.d/steam-dos/share/sounds/sf2`. From a flatpak app point of view, this is a system-wide path, so it has nothing to do with per-user xdg- dirs. Thus, both should be searched, I think.

Reply:
No, flatpak apps should bundle all the stuff they need (if it's not present in the runtime).

While replacing the list of well-know soundfonts dirs with `XDG_DATA_DIRS` is a good idea, searching in the steam-dos installation path is still needed.

Reply:
The point of this list is to find at least default soundfont file, so midi auto-detection would work out of the box. So far I assume that user read installation instructions and installed dependencies, but for flatpak that will not be the case. Do you think we should rather go into direction of bundling fluid soundfont with the steam-dos flatpak package or packaging it as a separate flatpak that would be pulled automatically (is it even possible? would it need to be part of a platform flatpak package?).

(This comment is not a blocker for code review, I will merge your changes later today unless they will fail in CI environment using python 3.5).

Reply:
Just to clarify: Flatpak maintains (almost) independent environment from the host, it's more like a container system than a packaging format. So, if soundfonts are installed on the host, flatpak apps can't access them.
>bundling fluid soundfont with the steam-dos flatpak

Yeah, this is necessary unless either Flatpak exposes host soundfonts to the apps, or the flatpak runtime includes them.

Reply:
Probably we can swith to using `XDG_DATA_DIRS` like this  gasinvein/steam-dos@b05defaf7efe864f8e5c7a5bd39f954156fbbf73:
https://github.com/dreamer/steam-dos/blob/b05defaf7efe864f8e5c7a5bd39f954156fbbf73/settings.py#L208-L226